Sources/StyleguideV2/HTMLHelpers/Link.swift

```diff
@@ -19,6 +19,7 @@ public struct Link<Label: HTML>: HTML {
   public var body: some HTML {
     a { label }
       .attribute("href", href)
+      .color(linkStyle.color)
```

Comment:
This is necessary to get Gmail to recognize link color, but it also causes a lot of churn in our snapshot tests. We could think about ways to limit this only to email html, but not sure how we want to do that.